pipeline/pipelines/grpc_generation_pipeline.py

```diff
@@ -97,6 +97,8 @@ def get_grpc_task_factory(language):
     cls = _GRPC_TASK_FACTORY_DICT.get(language)
     if cls:
         return cls()
+    elif language is None:
```

Comment:
note: typically you would just do this in the argparse configuration by making that flag required, but I actually prefer that you leave it this way because it will make transitioning execute pipeline from a script to an orchestrator easier when the artman wrapper is available.